megazords/full/src/lib.rs

```diff
@@ -25,6 +27,17 @@ pub use viaduct;
 // TODO: Uncomment this code when webext-storage component is integrated in android
 // pub use webext_storage;
 
+use nss::ensure_initialized;
+
+// TODO: would be nice if we could write
+//  #[uniffi::export]
+//  pub use nss::ensure_initialized as ensure_nss_initialized;
+// instead.
+#[uniffi::export]
+pub fn ensure_nss_initialized() {
+    ensure_initialized()
+}
+
```

Comment:
@bendk is there a way to do something like

```
#[uniffi::export]
pub use nss::ensure_initialized as ensure_nss_initialized;
```

or a way to avoid wrapping the function?

Reply:
There is https://mozilla.github.io/uniffi-rs/latest/proc_macro/functions.html#renaming-functions-methods-and-constructors, but I don't think that will actually work here :( So I don't think there is a good way, but it seems like a reasonable feature request!

Reply:
doesn't look like there's already an issue filed for this, will do that

Reply:
Thanks for filing that.  IMO, we could extend the concept of "external types" to also support "external functions".

Reply:
https://github.com/mozilla/uniffi-rs/issues/2461